Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -45,7 +45,7 @@ public enum RenderBlockContent: Equatable {
     /// A heading with the given level.
     case heading(level: Int, text: String, anchor: String?)
     /// A list that contains ordered items.
-    case orderedList(items: [ListItem])
+    case orderedList(start: Int?, items: [ListItem])
```

Comment:
Similar to the feedback I provided in Swift-Markdown, I'm wondering if this should be non-optional since all ordered lists have a start. Also, wondering if `startIndex` or `startingNumber` might be it clearer what this property refers to.